server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -730,6 +730,14 @@ timestamp_precision=ms
 # Datatype: long
 # slow_query_threshold=5000
 
+# Signal-noise-ratio (SNR) of FREQ encoding
+# Datatype: double
+# freq_snr=40.0
+
+# Block size of FREQ encoding
+# Datatype: integer
+# freq_block_size=1024
+
```

Comment:
Add these parameters to `Reference/Config-Manual.md`.

Reply:
Added